WooCommerce/Classes/POS/Presentation/PointOfSaleCollectCashView.swift

```diff
@@ -5,6 +5,7 @@ import WooFoundation
 struct PointOfSaleCollectCashView: View {
     @Environment(\.dynamicTypeSize) var dynamicTypeSize
     @Environment(\.posAnalytics) var analytics
+    @Environment(\.posExternalViews) var externalViews
```

Comment:
nit: these could be private:

```suggestion
    @Environment(\.dynamicTypeSize) private var dynamicTypeSize
    @Environment(\.posAnalytics) private var analytics
    @Environment(\.posExternalViews) private var externalViews
```